website/docs/help-and-support/troubleshooting-guide/js-errors.md

```diff
@@ -42,21 +46,21 @@ The solution to this is to bind the array inside the response object or transfor
 }
 ```
 
-To overcome this, we can bind the user's array of the response instead of the entire response object using javascript:
+To overcome this, you can bind the user's array of the response instead of the entire response object using JavaScript:
 
 ```javascript
 {{ fetch_users.data.users }}
 ```
 
-#### **This value does not evaluate to type Array\[{ label: string, value: string }]**
+**This value doesn't evaluate to type Array\[{`label: string, value: string`}]**
```

Comment:
This should look like line 15, starting with 

#### Error message
 ** This value doesn't evaluate to type Array\[{`label: string, value: string`}]** 

Same for line nos 86, 111 and 131

Reply:
I bolded the error messages as were looking like normal text. Once the action not triggered PR is merged, I'll change these to the message component.

Reply:
Yeah that's fine. Sorry for the confusion. I was asking to add the heading **Error message** before every error message like in screenshot below
<img width="797" alt="Screenshot 2022-12-12 at 18 35 26" src="https://user-images.githubusercontent.com/32227135/207052362-bdaa0162-f3ea-4d6d-8df4-99058d81988a.png">

Reply:
We discussed as this is all related to an array so only the array data type changes and will keep it highlighted but part of the same error message heading. The way we kept it in action was not triggered.

Reply:
This has been updated. so closing conversation.